<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureUi</class>
 <widget class="QWidget" name="ConfigureUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>UI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="general_groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_change_language_info">
          <property name="text">
           <string>Note: Changing language will apply your configuration.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="language_label">
            <property name="text">
             <string>Interface language:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="language_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="theme_label">
            <property name="text">
             <string>Theme:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="theme_combobox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GameListGroupBox">
     <property name="title">
      <string>Game List</string>
     </property>
     <layout class="QHBoxLayout" name="GameListHorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="GeneralVerticalLayout">
        <item>
         <widget class="QCheckBox" name="show_compat">
          <property name="text">
           <string>Show Compatibility List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_add_ons">
          <property name="text">
           <string>Show Add-Ons Column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_size">
          <property name="text">
           <string>Show Size Column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_types">
          <property name="text">
           <string>Show File Types Column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_play_time">
          <property name="text">
           <string>Show Play Time Column</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="game_icon_size_qhbox_layout_2">
          <item>
           <widget class="QLabel" name="game_icon_size_label">
            <property name="text">
             <string>Game Icon Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="game_icon_size_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="folder_icon_size_qhbox_layout_2">
          <item>
           <widget class="QLabel" name="folder_icon_size_label">
            <property name="text">
             <string>Folder Icon Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="folder_icon_size_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="row_1_qhbox_layout">
          <item>
           <widget class="QLabel" name="row_1_label">
            <property name="text">
             <string>Row 1 Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="row_1_text_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="row_2_qhbox_layout">
          <item>
           <widget class="QLabel" name="row_2_label">
            <property name="text">
             <string>Row 2 Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="row_2_text_combobox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="screenshots_GroupBox">
     <property name="title">
      <string>Screenshots</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="enable_screenshot_save_as">
          <property name="text">
           <string>Ask Where To Save Screenshots (Windows Only)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Screenshots Path: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="screenshot_path_edit"/>
          </item>
          <item>
           <widget class="QToolButton" name="screenshot_path_button">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="screenshot_width">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="screenshot_height">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Resolution:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
